seed/seed.go

```diff
@@ -61,6 +61,11 @@ func (s *Snap) ID() string {
 	return s.SideInfo.SnapID
 }
 
+// PlaceInfo returns a PlaceInfo for the seed snap.
+func (s *Snap) PlaceInfo() snap.PlaceInfo {
```

Comment:
Oh this is much nicer. I was worried about returning the "whole" `snap.Info` without potentially having all the data but `PlaceInfo` is more constrained.